CMakeLists.txt

```diff
@@ -33,6 +33,7 @@ option(BUILD_DAGMC_EXPORTER  "Include DagMC export capability."     ON )
 option(BUILD_MCNP_IMPORTER   "Include MCNP import capability."      ON )
 option(BUILD_IGEOM           "Include iGeom."                       OFF)
 option(BUILD_IGEOM_TESTS     "Include tests for iGeom."             OFF)
+option(CUBIT_17              "Using a version of Trelis or Cubit >= 17.0" OFF)
```

Comment:
I'm honestly not sure what version of Cubit this might apply to.

> Maybe there is an API number that we can refer to? It doesn't really matter except for clear communication to those who will try to build this.

I wish!! I looked in the `* .cmake` files of the 17.1 SDK for a version or API number but I didn't see one.

Reply:
Sorry, neither of those replies are particularly helpful. The only thing I can think of is to update the docstring on that option to indicate that this should be based on the SDK version being used as someone building the plugin is sure to know what version that is.

Reply:
:facepalm: They definitely do have a version header we can use to detect this. @ljacobson64 found it and is using it in #64.

Still doesn't quite answer the question about Cubit vs. Trelis Version numbering though...

Reply:
I don't think my PR has any relation to this specific conversation. The only difference between my PR and this one is that instead of telling users to specify whether their version of Trelis is >= 17, it tells users to specify their version of Trelis, and CMake internally determines whether that version is >= 17.

Reply:
My assumption about Cubit vs. Trelis version numbering is that they're the same. If you look in `/opt/Trelis-XX.X/include/util/CubitVersion.h`, it defines both `CUBIT_VERSION` and `TRELIS_VERSION` to the same thing

Reply:
Perhaps we could use `CubitVersion.h` to get the version?

Reply:
Ah, yeah I should've looked at #64 more closely. I assumed that was what you were doing, and, ideally, I think we probably should do just that -- detect the version from `CubitVersion.h`. That header is provided by the SDK, so we'd then be assuming that the SDK was matched with an appropriate version of Cubit/Trelis, but given that the headers defining the API are also provided with the SDK I think this is the best we can do.